adr/2022_02_11_declaring_services_in_container.md

```diff
@@ -0,0 +1,65 @@
+# Declaring services in container
+
+* Status: accepted
+* Date: 2022-02-11
+
+## Context and Problem Statement
+
+Since Symfony 3.4, all Symfony services are declared private by default. And this is the recommended Symfony approach. On the 
+other hand, first stable Sylius release was published before that and therefore at the moment of publication of this recommendation
+Sylius had thousands (or at least hundreds) public services. The question is how should we handle services declared after the 
```

Comment:
`thousands (or at least hundreds)` I think we can count that to be precise 😄 

<img width="752" alt="Zrzut ekranu 2022-02-11 o 16 28 51" src="https://user-images.githubusercontent.com/6212718/153619894-255b9f5a-c723-4579-ab22-940aa0abe086.png">

On 1.11 - 964 services with "sylius.", not aliases but these are both private and public... so I think we can safely write "hundreds" 😄